contents.json

```diff
@@ -6590,6 +6590,15 @@
       "date_added": "Wed, 1 Apr 2015 09:30:22 -0400",
       "suggested_by": "@jeffreyjackson"
     },
+    {
+            "title": "fitPlant",
+            "category-ids": ["fitness"],
+            "description": "Workout tracker where you can grow artificial plants",
+            "source": "https://github.com/KrisKodira/fitPlant",
+            "screenshots": ["https://github.com/KrisKodira/fitPlant/blob/master/design/ios-app/main-view.png"],
```

Comment:
@KrisKodira the link should be direct to the image. Could you fix that?

Reply:
Fixed it :) 👍